src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -378,6 +378,18 @@ doclet.help.search.refer=Refer to the {0} for a full description of search featu
 doclet.help.search.spec.url=https://docs.oracle.com/en/java/javase/{0}/docs/specs/javadoc/javadoc-search-spec.html
 # The title for the Javadoc Search Specification
 doclet.help.search.spec.title=Javadoc Search Specification
+doclet.help.releases.head=Release Details
+doclet.help.releases.body.specify.top-level=\
+    The details for each module, package, class or interface \
+    normally include the release in which the declaration was introduced.
+doclet.help.releases.body.specify.member=\
+    The details for each member of a class or interface normally \
+    include the release in which the member was introduced, in those situations \
+    where the member was added after the initial introduction of the \
+    enclosing class or interface.
+doclet.help.releases.body.refer=\
+    Some summary pages allow you to filter the contents of the page according to \
+    the release in which the declaration was introduced or deprecated.
```

Comment:
We already have help sections for New and Deprecated API, and the paragraph is somewhat vague. I think we should at least mention the summary pages by name ("New API", "Deprecated API"). Maybe link to the pages or help sections?

Reply:
The hard bit is that those pages may not always be present, depending on the command-line options...

Reply:
I still think this is not idea ("Some summary pages" then followed by "introduced or deprecated"), but it shouldn't be a show-stopper either. It's better to have this covered, even if the wording is not perfect.